src/App.jsx

```diff
@@ -151,19 +162,16 @@ function App() {
                 data={currentWeatherData.pressure}
                 unit={"mb"}
               />
-            </div>
+            </MainHighlight>
 
-            <a
-              href="https://github.com/bruhGrassi"
-              target="_blank"
-              className="link"
-            >
+            <Link href="https://github.com/bruhGrassi" target="_blank">
               Create by <span>Bruna Grassi</span>
-            </a>
-          </main>
-        </section>
+            </Link>
+          </Main>
+        </Wrapper>
       )}
-    </div>
+      <GlobalStyles />
```

Comment:
I'd have the `GlobalStyles` at the top, even out of Container and using a `<>` (fragment) to encapsulate them.

Ps.: this is mostly nitpick, it's okay to put it anywhere actually.